content/chainguard/libraries/overview.md

```diff
@@ -57,27 +58,26 @@ upstream source of a project is available in a repository. In addition, these
 repositories also hold non-open source binaries of libraries.
 
 All these factors create uncertainty. Using these public repositories can feel
-as opaque as picking up a USB drive off of the sidewalk and plugging its
-contents into our production environment.
+as opaque as picking up a USB drive off of the sidewalk and plugging it into your laptop.
 
-## Introduction 
+## Introduction
 
 Chainguard Libraries builds all available libraries from source code in the
 Chainguard Factory and makes them available for you. The Chainguard Factory
-represents Chainguard's internal tooling that enables a more secure, dedicated,
+is Chainguard's internal tooling that enables a more secure, dedicated,
 private, and SLSA-certified build infrastructure for building software from
 source and publishing the binaries to customers.
 
-Chainguard Libraries and the use of the Chainguard Factory remove all software
+Chainguard Libraries and the use of the Chainguard Factory remove many software
 supply chain problems for libraries:
 
-* Build stage protection - All binary libraries and library versions are built
+* Build stage protection: All binary libraries and library versions are built
```

Comment:
```suggestion
* All binary libraries and library versions are built
```

If you're not going to use the `THING: EXPLANATION` format for each bullet I wouldn't use it at all